packages/app-store/paypal/components/EventTypeAppCardInterface.tsx

```diff
@@ -29,6 +29,15 @@
     isAcceptedCurrencyCode(currency) ? currencySymbols[currency] : ""
   );
 
+  let credentialId: number;
```

Comment:
Same here

Reply:
Fixed in this commit https://github.com/calcom/cal.com/commit/b88334b0cd6c8b92ba7d2870c61e8797eb7ecbd3